scheduler/src/cook/mesos/api.clj

```diff
@@ -1916,6 +1986,30 @@
                       :parameters {:query-params JobOrInstanceIds}
                       :handler (destroy-jobs-handler conn is-authorized-fn)}}))
 
+        (c-api/context
+          "/jobs/:uuid" [uuid]
+          :path-params [uuid :- s/Uuid]
+          (c-api/resource
+            {:get {:summary "Returns info about a single Job"
+                   :responses {200 {:schema JobResponse
+                                    :description "The job was returned."}
+                               400 {:description "A non-UUID value was passed."}
+                               403 {:description "The supplied UUID doesn't correspond to a valid job."}}
+                   :handler (read-jobs-handler-single conn framework-id is-authorized-fn
+                                                      retrieve-sandbox-directory-from-agent)}}))
+
+        (c-api/context
+          "/jobs" []
+          (c-api/resource
+            {:get {:summary "Returns info about a set of Jobs"
```

Comment:
Nitpick: Should the summary mention the name of the query parameter to use?

Reply:
We aren't doing this in the other summary fields. The swagger UI shows parameters separately:
![image](https://user-images.githubusercontent.com/444778/33095640-f2444a16-cec9-11e7-97c5-f624557f877a.png)